apps/monitoring/README.md

```diff
@@ -0,0 +1,65 @@
+# Saleor Monitoring app
+
+## Local development
+
+### Start Monitoring backend
+Run:
+ ```shell
+docker-compose up
+ ```
+It is beneficial to run this command in a separate terminal tab to observe backend logs easily.
+
+By default, backend will run at `localhost:5001` with:
+   - Manifest at `/manifest`
+   - Graphql Playground at `/graphql`
+   - OpenApi viewer at `/docs`
+
+### Develop frontend:
+
+Frontend app is located at:
+```shell
+cd frontend
+```
+Installing dependencies with:
+```shell
+pnpm i
+```
+Running dev server
+```shell
+npm run dev
+```
+The frontend app will run at `localhost:3000`.
```

Comment:
To be considered - since this app will most likely run alongside other apps, maybe it would be a good idea to use a different default port? 
Probably it's a bigger discussion if we want to change ports in other apps too

Reply:
I think its likely that you develop one app at the time - so you run `pnpm dev --filter=saleor-app-x`

Then, you always have 3000, and if you run other apps, you need to find what port was assigned

If you set port ahead of time, you will need to learn ports for all apps, no matter how many are running

I'd probably leave it like it is, but maybe we can configure some dynamic proxy that will route domains, eg `http://saleor-app-invoices.local` -> `localhost:3005`